PWG/EMCAL/EMCALtasks/AliAodSkimTask.cxx

```diff
@@ -171,6 +171,23 @@ void AliAodSkimTask::UserExec(Option_t *)
     storePt = kTRUE;
   }
 
+  // Accept event only if an track within a pT interval (fTrackMinPt, fTrackMaxPt) has been found in the event
```

Comment:
I would have preferred a solution where we have one loop that decides all cases depending on the value of fTrackMinPt and fTrackMaxPt (ie only check the dependence if they are different from -1)

Reply:
OK i will change the code